applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -287,18 +303,24 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 		for (int j = 0; j < base->subgroups[i].bis_count; j++) {
 			const uint8_t index = base->subgroups[i].bis_data[j].index;
 
-			LOG_DBG("BIS %d   index = %hhu", j, index);
+			LOG_DBG("Subgroup: %d BIS: %d index = %d", i, j, index);
 
 			if (bitrate_check((struct bt_codec *)&base->subgroups[i].codec)) {
 				suitable_stream_found = true;
 
 				bis_index_bitfields[sync_stream_cnt] = BIT(index);
 
+				/* Get general (level 2) codec config from the subgroup*/
 				audio_streams[sync_stream_cnt].codec =
 					(struct bt_codec *)&base->subgroups[i].codec;
 				get_codec_info(audio_streams[sync_stream_cnt].codec,
 					       &audio_codec_info[sync_stream_cnt]);
-				print_codec(&audio_codec_info[sync_stream_cnt]);
+
+				/* Overwrite codec config with level 3 BIS specific codec config.
+				 * For now, this is only done for channel allocation
+				 */
+				bis_specific_codec_config(base->subgroups[i].bis_data[j],
```

Comment:
Return code is not checked here

Reply:
That was on purpose. I have added a (void) here to make it clear.